instance/models/mixins/storage.py

```diff
@@ -231,7 +232,7 @@ def get_s3_connection(self):
             self.s3_secret_access_key
         )
 
-    def _create_bucket(self, attempts=4, ongoing_attempt=1, retry_delay=1):
+    def _create_bucket(self, attempts=4, ongoing_attempt=1, retry_delay=2):
```

Comment:
Have you measured how long we typically need to wait?  If 4 attempts with a one-second interval are not enough, wouldn't it be better to at least increse the delay to 5 seconds?

Reply:
I will deploy to staging now, and test.

Reply:
tested on staging, went through without any retry.